packages/lib/i18n.ts

```diff
@@ -12,3 +12,28 @@ export const localeOptions = locales.map((locale) => ({
 export const defaultLocaleOption = localeOptions.find(
   (locale) => locale.value === i18n.defaultLocale
 ) as (typeof localeOptions)[number];
+
+interface ExtendedLocale extends Intl.Locale {
+  getTextInfo?: () => { direction: "ltr" | "rtl" };
+}
```

Comment:
I don't see getTextInfo working anywhere, it's just mentioned in the docs so it's best to just remove this and use .textInfo.direction which works.